tests/test.yml

```diff
@@ -5,7 +5,7 @@
    vars:
      - dell_dsu_repo_install: True
      - dell_dsu_update_all_firmware: False
-     - dell_dsu_arguments: "--non-interactive --apply-upgrades-only --preview"
+     - dell_dsu_arguments: "--non-interactive --apply-upgrades --preview"
```

Comment:
In a way this PR is changing the role's "user interface" from

`dell_dsu_arguments`

to the tuple of

`dell_dsu_arguments_base`
`upgrade_categories`

thus I think the tests should also be modified to utilize at least the former, if not also the latter. (E.g. default to something simple like `upgrade_categories="iDRAC"`)

Naturally the Travis Docker container in which the test runs doesn't have Dell hardware, but I suppose that shouldn't matter when running `--preview`.